### PR TITLE
Rebenchmark mantaSbt

### LINKED ISSUE #1151: Rebenchmark pallet mantaSbt

### PR DESCRIPTION
**Pull request by Dengjianping**
## Description

Closed #1151 

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: tests/mantaSbt_toprivate.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    });
+    await api.isReady;
+    return api;
+}
+
+async function main() {
+    const api = await createPromiseApi(nodeAddress);
+    const sender = keyring.addFromMnemonic("bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice");
+
+    const mints_file = '/home/jamie/my-repo/Manta/pallets/manta-sbt/precomputed_mints-0-to_20';
```
Comment (ghzlatarev) [inline tests/mantaSbt_toprivate.js]: better relative path, maybe to the data folder

**Review by ghzlatarev — COMMENTED** [review]
Seems good, I wonder what the weights results will be

**Review by ghzlatarev — APPROVED** [review]

File: tests/append_storage.js [hunk 2]
```diff
@@ -7,19 +8,37 @@ let prefixes = [
...
+
 async function main() {
   let storagePath = './data/storage.json';
+  let mantaSbtStoragePath = './data/mantaSbtStorage.json';
   let forkPath = './data/fork.json';
 
   let storage = JSON.parse(fs.readFileSync(storagePath, 'utf8'));
   let forkedSpec = JSON.parse(fs.readFileSync(forkPath, 'utf8'));
-
+
```
Comment (ferrell-code) [inline tests/append_storage.js:21]: nit: extra space

**Review by ferrell-code — APPROVED** [review]
nice

**Review by ghzlatarev — APPROVED** [review]

**Review by anaye1997 — APPROVED** [review]

**Pull request merged**